ibis/expr/api.py

```diff
@@ -4032,6 +4032,53 @@ def _table_count(self):
     return ops.Count(self, None).to_expr().name('count')
 
 
+def _table_dropna(self, how='any', subset=None):
+    """
+    Remove rows with null values from the table.
+
+    Parameters
+    ----------
+    how : determine whether a row is removed if there is at least one null
+    value in the row ('any'), or if all row values are null ('all').
+    Options are 'any' or 'all'. Default is 'all'
+    subset : Optional list of columns names to consider.
+    Defaults to all columns.
+
+    Returns
+    -------
+    table : TableExpr
+      New table expression
+    """
+    if how not in {'any', 'all'}:
+        raise ValueError("Please specify how as either 'any' or 'all'.")
+    subset = util.promote_list(subset)
+    invalid = set(subset) - set(self.schema().names)
+    if invalid:
+        raise ValueError(f'Columns {list(invalid)} do not exist in the table.')
+    return ops.DropNa(self, how, subset).to_expr()
```

Comment:
We should implement this for at least one SQL backend in a follow up. It can likely be done as a rewrite along the lines of:

```python
from operator import and_, or_
from functools import reduce

t[reduce(or_ if how == 'any' else and_, subset)]
```

Reply:
Created an issue to track https://github.com/ibis-project/ibis/issues/3110